src/Entity/DigitalPost.php

```diff
@@ -238,4 +253,40 @@ public function updateAttachmentPositions()
             $attachment->setPosition($index++);
         }
     }
+
+    public function getSubject(): ?string
+    {
+        return $this->subject;
+    }
+
+    public function setSubject(string $subject): self
+    {
+        $this->subject = $subject;
+
+        return $this;
+    }
+
+    public function getNext(): ?self
+    {
+        return $this->next;
+    }
+
+    public function setNext(?self $next): self
+    {
+        $this->next = $next;
```

Comment:
This should probably also set `$this->previous` (setting just one doesn't make sense).

Reply:
Updated linking logic.